samples/bluetooth/mesh/light/CMakeLists.txt

```diff
@@ -22,5 +22,8 @@ target_include_directories(app PRIVATE include)
 target_sources_ifdef(CONFIG_BOARD_THINGY53_NRF5340_CPUAPP app PRIVATE
   boards/thingy53.c
 )
+target_sources_ifdef(CONFIG_BOARD_THINGY53_NRF5340_CPUAPP_NS app PRIVATE
```

Comment:
Maybe
```
if (CONFIG_BOARD_THINGY53_NRF5340_CPUAPP OR CONFIG_BOARD_THINGY53_NRF5340_CPUAPP_NS)
   target_sources...
endif()
```
Also for other samples

Reply:
It's already done like this for board related files in Zephyr

Reply:
I don't mind it being as it is now. It's clear to distinguish board support and provides little help for anything else.